cmd/gtfToBed/gtfToBed.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/common"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+	"strings"
+)
+
+func gtfToBed(fileName string, outFile string) {
+	file := fileio.EasyOpen(fileName)
+	defer file.Close()
+
+	out := fileio.EasyCreate(outFile)
+	defer out.Close()
+
+	var line string
+	var annotationStringSlice []string
+	var currBed bed.Bed
+	var doneReading bool = false
+
+	for line, doneReading = fileio.EasyNextRealLine(file); !doneReading; line, doneReading = fileio.EasyNextRealLine(file) {
+		words := strings.Split(line, "\t")
+		annotationStringSlice = make([]string, 1) //annotation contains all fields except the position information and name, so four fields are not used here.
+		annotationStringSlice[0] = words[1]
```

Comment:
I think this is a bigger question, but maybe we should think about keep bed to be strictly a bed and then creating a new file format/struct that is similar to bed, but allows fields of free annotation after the first few?  I am worried that we have invented our own file format that is similar to beds, but not a bed, yet we are calling it a bed.